ffjson.go

```diff
@@ -52,12 +53,61 @@ func main() {
 	}
 
 	inputPath := filepath.ToSlash(extra[0])
+	inputFiles := make([]string, 0)
 
 	var outputPath string
-	if outputPathFlag == nil || *outputPathFlag == "" {
-		outputPath = extRe.ReplaceAllString(inputPath, "${1}_ffjson.go")
+	if extRe.MatchString(inputPath) {
+		if outputPathFlag == nil || *outputPathFlag == "" {
+			outputPath = extRe.ReplaceAllString(inputPath, "${1}_ffjson.go")
+		} else {
+			outputPath = *outputPathFlag
+		}
+		inputFiles = []string{inputPath}
 	} else {
-		outputPath = *outputPathFlag
+		// No extension, try package
+		if inputPath == "." {
```

Comment:
this could just be a full package path... eg, `github.com/pquerna/mything` rather than just `.` -- we could just see if `inputPath` is a file, if not, try to treat it like a directory.